BookingsApi/BookingsApi/Controllers/V1/PersonsController.cs

```diff
@@ -350,5 +350,34 @@ public async Task<IActionResult> UpdatePersonUsername(string contactEmail, strin
 
             return NoContent();
         }
+        
+        /// <summary>
+        /// For usage by automation tests, to delete test users
+        /// </summary>
+        /// <param name="username"></param>
+        /// <returns></returns>
+        [HttpDelete("{username}")]
```

Comment:
just wondering if it's worth changing the route to `[HttpDelete("test-persons/{username}")]` to match the operation name (or moving to a TestPersonsController). Also avoids a name clash if we do ever implement deleting of actual persons

Reply:
Yeah that makes sense. The controller originally was just called DeletePerson, but I went back and updated it later as I felt it should only be used for that one task of data clean up. Will update the routing now.